client/src/main.rs

```diff
@@ -281,22 +281,6 @@ enum Command {
     },
 }
 
-/// Application-level error.
```

Comment:
This was a clippy warning from the lastest 1.78.0

Reply:
What was the clippy warning?

Reply:
```
warning: struct `ClientError` is never constructed
   --> client/src/main.rs:286:19
    |
286 | pub(crate) struct ClientError(String);
    |                   ^^^^^^^^^^^
    |
    = note: `#[warn(dead_code)]` on by default
```